app/src/main/java/org/oppia/android/app/topic/TopicTab.kt

```diff
@@ -33,7 +33,7 @@ enum class TopicTab(
     positionWithTwoTabs = 1,
     positionWithFourTabs = 3,
     tabLabelResId = R.string.revision,
-    tabIconResId = R.drawable.ic_revision_icon_24dp
```

Comment:
Do we use ic_revision_icon_24dp elsewhere? We should be replacing the existing drawable itself rather than introducing a new one.

Reply:
Resolved in the latest changes.